js/addToTask.js

```diff
@@ -9,14 +9,15 @@ import {
   TODAY,
   COMPLETE,
   EDIT,
-  DELET,
+  DELETE,
```

Comment:
constant name is too generic, try to rename it to be domain specific i.e `DELETE_TODO`